client/src/includes/initIconSprite.ts

```diff
@@ -0,0 +1,45 @@
+export const initIconSprite = (
+  spriteContainer: HTMLElement,
+  spriteURL: string,
+) => {
+  const revisionKey = 'wagtail:spriteRevision';
+  const dataKey = 'wagtail:spriteData';
```

Comment:
Let's put these two `keys` into the function param above, makes no functional difference but let's us treat these as data that's coming into the function.

Also, not critical but it's nice to say the function returns `void` to tell TypeScript that the function will not return anything.

```suggestion
  revisionKey = 'wagtail:spriteRevision',
  dataKey = 'wagtail:spriteData',
): void => {
```